cluster/addons/podsecuritypolicies/restricted.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: restricted-v1
+  annotations:
+    kubernetes.io/description: 'This policy demonstrates best practices for
```

Comment:
Consider adding `requiredDropCapabilities` to `restricted-v1` PSP in order to conform to best practices:
```yaml
requiredDropCapabilities:
- KILL
- MKNOD
- SETUID
- SETGID
```

Reply:
Hmm, good suggestion, except there doesn't appear to be any way to default the dropped capabilities list. I'm not an advocate of the PSP defaults, but specifying this as is in the restricted profile would require every pod to explicitly list these 4 dropped capabilities....

WDYT?

Reply:
Oops, nvm. I realized `requiredDropCapabilities` is mutating.

In that case, I dropped a lot more, mostly following this guide: http://rhelblog.redhat.com/2016/10/17/secure-your-containers-with-this-one-weird-trick/

Open for discussion, but if there is too much back-and-forth, I might save this for a follow up PR.

Reply:
> Open for discussion, but if there is too much back-and-forth, I might save this for a follow up PR.

Yes, it's better to discuss it separately. In OpenShift we're dropping only these 4 caps (`KILL`, `MKNOD`, `SETUID`, `SETGID`) in the `restricted` SCC.